sound/soc/intel/boards/Makefile

```diff
@@ -41,10 +41,10 @@ snd-soc-sof-sdw-objs += sof_sdw.o				\
 			sof_sdw_rt5682.o sof_sdw_rt700.o	\
 			sof_sdw_rt711.o sof_sdw_rt_sdca_jack_common.o	\
 			sof_sdw_rt712_sdca.o sof_sdw_rt715.o	\
-			sof_sdw_rt715_sdca.o sof_sdw_dmic.o	\
+			sof_sdw_rt715_sdca.o sof_sdw_ssp_dmic.o	\
```

Comment:
this is too confusing, the ssp and dmic have nothing in common, this rename is really weird and not rooted in any technical reality. We can have dmic without ssp.

Reply:
restored